.github/workflows/coverity.yml

```diff
@@ -0,0 +1,30 @@
+name: libbpf-ci-coverity
+
+on:
+  schedule:
+    - cron:  '0 18 * * *'
+
+
+jobs:
+  coverity:
+    runs-on: ubuntu-latest
+    name: Coverity
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/setup
+      - name: Run coverity
+        run: |
+          echo ::group::Setup CI env
+          source /tmp/ci_setup
+          export COVERITY_SCAN_NOTIFICATION_EMAIL="${AUTHOR_EMAIL}"
+          export COVERITY_SCAN_BRANCH_PATTERN=${GITHUB_REF##refs/*/}
+          export TRAVIS_BRANCH=${COVERITY_SCAN_BRANCH_PATTERN}
+          echo ::endgroup
+          scripts/coverity.sh
+        env:
+          COVERITY_SCAN_TOKEN: ${{ secrets.COVERITY_SCAN_TOKEN }}
+          COVERITY_SCAN_PROJECT_NAME: libbpf
+          COVERITY_SCAN_BUILD_COMMAND_PREPEND: 'cd src/'
+          COVERITY_SCAN_BUILD_COMMAND: 'make'
+      - name: SCM log
+        run: cat /home/runner/work/libbpf/libbpf/src/cov-int/scm_log.txt
```

Comment:
seems like there are a lot of unrelated warnings, do you think we can avoid them somehow? Coverity has ability to specify file patterns to ignore or categorize errors, it could be a way to make those logs clean. Either way, just something I noticed (I didn't see those scm_logs.txt before).

Reply:
I've seen them and decided not to investigate too deep as it's orthogonal to GH actions migration - it seems like we have same behavior for travis, nobody just bothered looking at this log.